src/config/datatables.php

```diff
@@ -39,9 +39,10 @@
      * This is where you can register your custom dataTables builder.
      */
     'engines' => [
-        'eloquent'   => \Yajra\DataTables\EloquentDataTable::class,
-        'query'      => \Yajra\DataTables\QueryDataTable::class,
-        'collection' => \Yajra\DataTables\CollectionDataTable::class,
+        'eloquent'                    => \Yajra\DataTables\EloquentDataTable::class,
+        'query'                       => \Yajra\DataTables\QueryDataTable::class,
+        'collection'                  => \Yajra\DataTables\CollectionDataTable::class,
+        'anonymousresourcecollection' => \Yajra\DataTables\ApiResourceDataTable::class,
```

Comment:
Maybe we use shorter syntax like `resource`? Also this matches, `public function resource($resource)`.